src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -441,16 +496,33 @@ public Void call() throws Exception {
                             );
                             break;
                         }
+                        if (previousLine != null) {
+                            sb.append(previousLine).append(System.getProperty("line.separator"));
```

Comment:
@dean-e-clark `System.getProperty("line.separator")` is duplicated, can we extract it to a variable?